scripts/prepare_cluster.sh

```diff
@@ -222,11 +226,25 @@ for ((i=0; i<${#MASTERS[@]}; i++)); do
         yq e 'select(.kind == "CustomResourceDefinition")' "$OUTPUT_DIR_MASTER/root/manifest-kube-system.yaml" >> "$OUTPUT_DIR_MASTER/root/crds.yaml"
     fi
 
+    if [ $IS_IN_AZURE == "true" ]; then
+        unique_fourth_octet=$(echo $NODE_HOST_IP | cut -d . -f 4)
+        if [ -n "$AZURE_VXLAN_IPRANGE" ]; then
+            newrange=${AZURE_VXLAN_IPRANGE%.*}.$unique_fourth_octet/${AZURE_VXLAN_IPRANGE#*/}
+            # newrange=$(sed  -n 's/0.24/2/p' <<<$AZURE_VXLAN_IPRANGE)
+            export VXLAN_IP=$newrange
+        else
+            export VXLAN_IP="10.0.98.$unique_ip/24"
+        fi
```

Comment:
I think we should just add the default value at line 66, ex:
```
AZURE_VXLAN_IPRANGE=${AZURE_VXLAN_IPRANGE:-10.0.98.0/24}
```
and then simplify this to:

```suggestion
        newrange=${AZURE_VXLAN_IPRANGE%.*}.$unique_fourth_octet/${AZURE_VXLAN_IPRANGE#*/}
        export VXLAN_IP=$newrange
```